src/NumericalAlgorithms/FiniteDifference/Minmod.cpp

```diff
@@ -22,8 +23,25 @@ namespace fd::reconstruction::detail {
       const Index<DIM(data)>& volume_extents,                                  \
       const size_t number_of_variables) noexcept;
 
+namespace detail {
```

Comment:
Macros are part of the preprocessor, so their definition doesn't need to be in any namespace. However, `GENERATE_INSTANTIATIONS` calls a macro, which then actually injects code and so that code needs to be in the correct namespace